### ISSUE TITLE
Where are internal pullup resistors being enabled?  

### ISSUE THREAD

**Issue by synaption** [google-coral/edgetpu#748]
### Description

I am using the coral som with gpio 6,7, and 8, and at it seems as though the internal pullups are being activated at boot, but I cannot find where that is happening in the device tree.  My understanding is that the pulldown is enabled by default, and the pullup would have to be explicitly enabled with IOMUXC_SW_PAD_CTL in the device tree, but I do not see that anywhere.  

<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Others

### Operating System

Linux

### Programming Language

Other

### Relevant Log Output

_No response_</details>

**Comment by hjonnala** [comment]
I think, here is no control bit for pull down enable/disable for NXP IMX8M Soc. Please check with NXP community regarding this issue. Thanks!!

Ref: https://community.nxp.com/t5/i-MX-Processors/i-MX8M-processor-pins-pull-up-down-configuration/m-p/882224#M133754

**Comment by synaption** [comment]
Right.  This is what I was referencing also.  "There is internal 90K Ohm pull down resistor on the IOs to avoid excessive current through the resistive voltage"  this is enabled by default.  I'm asking about the pullup resistors not the pull down resistors.  "For the PUE bit field, it only controls the internal 27K Ohm pull up resistor enable/disable.  Customers can use IOMUXC registers, such as for example IOMUXC_SW_PAD_CTL_PAD_GPIO1_IO02 to define what pins have internal 27K Ohm pull up resistor enabled."  So it seems like the pullup is enabled, but I don't see any reference to IOMUXC_SW_PAD_CTL anywhere.  I'm trying to find where it is enabled so I can disable it.

**Comment by hjonnala** [comment]
I am not sure, if it helps. I can find those references in im-atf folder in [mendel source tree](https://coral.googlesource.com/docs/+/refs/heads/master/GettingStarted.md).  https://coral.googlesource.com/imx-atf/+/refs/heads/master/plat/imx/common/include/imx_io_mux.h#214

**Comment by synaption** [comment]
We're getting somewhere.  If I understand correctly the OFFSET defines the location of the bits that would be set, but does not set them.

**Comment by synaption** [comment]
Could it be set somewhere in [python-periphery](https://pypi.org/project/python-periphery/)?

**Comment by hjonnala** [comment]
Please check with NXP community if they have any suggestions on this issue. Thanks!

**Comment by easy-and-simple** [comment]
And how you know that internal pull-ups are enabled, and what is the problem if they are?
